applications/serial_lte_modem/Kconfig

```diff
@@ -53,20 +53,23 @@ config SLM_UART_RX_BUF_COUNT
 	default 3
 	help
 	  Amount of buffers for receiving (RX) UART traffic. If the buffers are full, UART RX will be disabled until the buffers are processed.
```

Comment:
This line is huge and will go off the screen of users, should be wrapped at 100 characters

Reply:
Thanks.
I split those lines.